packages/svelteui-core/src/components/Chip/Chip.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+	import useStyles from './Chip.styles.js';
+	import { randomID } from '$lib/styles';
+	import { get_current_component } from 'svelte/internal';
+	import { createEventForwarder, useActions } from '$lib/internal';
+	import Box from '../Box/Box.svelte';
+	import type { ChipProps as $$ChipProps } from './Chip.styles.js';
+
+	interface $$Props extends $$ChipProps {}
+
+	export let use: $$Props['use'] = [],
+		element: $$Props['element'] = undefined,
+		className: $$Props['className'] = '',
+		override: $$Props['override'] = {},
+		color: $$Props['color'] = 'blue',
+		id: $$Props['id'] = randomID(),
+		disabled: $$Props['disabled'] = false,
+		value: $$Props['value'] = null,
+		checked: $$Props['checked'] = false,
+		label: $$Props['label'] = null,
+		radius: $$Props['radius'] = 'sm',
+    size: $$Props['size'] = 'md',
+    variant: $$Props['variant'] = 'outline',
+    // type: $$Props['type'] = 'checkbox',
+		transitionDuration: $$Props['transitionDuration'] = 100;
+	export { className as class };
+
+	/** An action that forwards inner dom node events from parent component */
+	const forwardEvents = createEventForwarder(get_current_component());
+
+	$: ({ cx, classes, getStyles } = useStyles({ color, radius, size, transitionDuration, variant }));
+</script>
+
+<!--
+@component
+
+A picker for one or more options.
+
+@see https://svelteui.org/core/chip
+@example
+    ```svelte
+    <Chip>Chips</Chip>
+    <Chip size={'lg'}>Big Chip</Chip>
+    <Chip>Another one</Chip>
+    <Chip checked disabled>Disabled</Chip>
+    ```
+-->
+
+<Box bind:element class={cx('chip', className, getStyles({ css: override }))} {...$$restProps}>
+	<div class={classes.inner}>
```

Comment:
There seems to be no inner class, dead code? Perhaps the class `input` could be added here instead of in the input component (with another name perhaps), in the current state the input is taking space in the DOM

Reply:
You can put the input class where the inner is now, keeping the div

Reply:
Okay I've done that and it seemed to work. I've set the input to `visibility: hidden` so that it doesn't appear in the DOM.